test/Microsoft.ML.Predictor.Tests/ResultProcessor/TestData/MultiClassifier/0.out.txt

```diff
@@ -1,4 +1,4 @@
-maml.exe TrainTest test=F:\data\MNIST\Train-28x28.txt tr=MultiClassLogisticRegression{l1=0.1 initwts=0.5} loader=TextLoader{col=Label:R4:0 col=Features:R4:1-784} data=F:\data\MNIST\Test-28x28.txt
+maml.exe TrainTest test=F:\data\MNIST\Train-28x28.txt tr=MulticlassLogisticRegression{l1=0.1 initwts=0.5} loader=TextLoader{col=Label:R4:0 col=Features:R4:1-784} data=F:\data\MNIST\Test-28x28.txt
```

Comment:
Command-line breaker... But I feel it's fine.

Reply:
I think command line is case insensitive, but I can be wrong

---
In reply to: [265270749](https://github.com/dotnet/machinelearning/pull/2919#discussion_r265270749) [](ancestors = 265270749)